### PR TITLE
Remove Kustomize integration

### LINKED ISSUE #1226: Figure out how to run the kustomize tests on Windows
This is the only remaining issue from #1174

### LINKED ISSUE #1241: Remove Kustomize integration
It is not used right now and causes issues such as #1226.

### PR DESCRIPTION
**Pull request by fbiville**
Fixes #1226.
Fixes #1241.

### CODE REVIEW

File: pkg/riff/commands/wiring.go [hunk 1]
```diff
@@ -19,13 +19,11 @@ package commands
 import (
 	"context"
 	"fmt"
+	"github.com/pivotal/go-ape/pkg/furl"
```
[the remark this answers is not in the document]
Comment (fbiville) [reply pkg/riff/commands/wiring.go]: that's been moved there automatically by Goland :|

[the remark this answers is not in the document]
Comment (scothis) [reply pkg/riff/commands/wiring.go]: The newline before the github imports was probably removed which caused them to fall into the  group and be sorted.

**Comment by fbiville** [comment]
A fix has been found: https://github.com/projectriff/riff/pull/1247.

**Comment by scothis** [comment]
Closing in favor of #1247

**Pull request closed without merging**